packages/create-burner/src/manager/burnerManager.ts

```diff
@@ -208,6 +217,10 @@ export class BurnerManager {
     }
 
     public async create(): Promise<Account> {
+        if (!this.isInitialized) {
```

Comment:
this could put into a helper to keep code DRY as it's repeated

Reply:
It's not exactly the same condition. For the `create` method we trigger an error if not initialized and for the `init` method if it's already been initialized